stable/fluent-bit/README.md

```diff
@@ -29,16 +29,20 @@ The following table lists the configurable parameters of the Fluent-Bit chart an
 
 | Parameter                  | Description                        | Default                 |
 | -----------------------    | ---------------------------------- | ----------------------- |
-| **Backend Selection**      |
-| `backend.type`             | Set the backend to which Fluent-Bit should flush the information it gathers | `forward` |
+| **file Backend**        |
+| `backend.file.enabled`     | Boolean flag to indicate whether to enable the "file" output plugin | `fluentd` |
+| **stdout Backend**        |
+| `backend.stdout.enabled`     | Boolean flag to indicate whether to enable the "stdout" output plugin | `fluentd` |
```

Comment:
```suggestion
| `backend.stdout.enabled`     | Boolean flag to indicate whether to enable the "stdout" output plugin | `true` |
```
It's enabled by default, so the default value should be `true`, I guess. And similarly it should be `false` for other backends here.